*** /home/oleg/.local/share/openxcom/standard/xcom1/regions.rul	2017-04-16 18:49:14.000000000 +0300
--- regions.rul	2018-11-29 23:54:21.000000000 +0300
***************
*** 1,4 ****
--- 1,20 ----
  regions:
+   - delete: STR_NORTH_AMERICA
+   - delete: STR_ARCTIC
+   - delete: STR_ANTARCTICA
+   - delete: STR_SOUTH_AMERICA
+   - delete: STR_EUROPE
+   - delete: STR_NORTH_AFRICA
+   - delete: STR_SOUTHERN_AFRICA
+   - delete: STR_CENTRAL_ASIA
+   - delete: STR_SOUTH_EAST_ASIA
+   - delete: STR_SIBERIA
+   - delete: STR_AUSTRALASIA
+   - delete: STR_PACIFIC
+   - delete: STR_NORTH_ATLANTIC
+   - delete: STR_SOUTH_ATLANTIC
+   - delete: STR_INDIAN_OCEAN
+ 
    - type: STR_NORTH_AMERICA
      cost: 800000
      areas:
***************
*** 7,17 ****
        - [240, 300, -30, -10]
      regionWeight: 18
      missionWeights:
!       STR_ALIEN_RESEARCH: 14
!       STR_ALIEN_HARVEST: 17
!       STR_ALIEN_ABDUCTION: 20
!       STR_ALIEN_INFILTRATION: 20
!       STR_ALIEN_BASE: 20
      missionZones:
        -
          - [200, 220, -65, -60]
--- 23,29 ----
        - [240, 300, -30, -10]
      regionWeight: 18
      missionWeights:
!        #STR_ALIEN_FLYBY: 1
      missionZones:
        -
          - [200, 220, -65, -60]
***************
*** 84,91 ****
        - [0, 360, -90, -70]
      regionWeight: 4
      missionWeights:
!       STR_ALIEN_RESEARCH: 25
!       STR_ALIEN_BASE: 35
      missionZones:
        -
          - [305.25, 333.125, -79.25, -75.25]
--- 96,102 ----
        - [0, 360, -90, -70]
      regionWeight: 4
      missionWeights:
!        #STR_ALIEN_FLYBY: 1
      missionZones:
        -
          - [305.25, 333.125, -79.25, -75.25]
***************
*** 135,142 ****
        - [0, 360, 60, 90]
      regionWeight: 4
      missionWeights:
!       STR_ALIEN_RESEARCH: 25
!       STR_ALIEN_BASE: 35
      missionZones:
        -
          - [77, 108.125, 62.125, 76.125]
--- 146,152 ----
        - [0, 360, 60, 90]
      regionWeight: 4
      missionWeights:
!        #STR_ALIEN_FLYBY: 1
      missionZones:
        -
          - [77, 108.125, 62.125, 76.125]
***************
*** 184,194 ****
        - [275, 325, 15, 60]
      regionWeight: 10
      missionWeights:
!       STR_ALIEN_RESEARCH: 10
!       STR_ALIEN_HARVEST: 24
!       STR_ALIEN_ABDUCTION: 12
!       STR_ALIEN_INFILTRATION: 18
!       STR_ALIEN_BASE: 22
      missionZones:
        -
          - [280.375, 297.625, -11.875, -4.125]
--- 194,200 ----
        - [275, 325, 15, 60]
      regionWeight: 10
      missionWeights:
!        #STR_ALIEN_FLYBY: 1
      missionZones:
        -
          - [280.375, 297.625, -11.875, -4.125]
***************
*** 239,249 ****
        - [335, 60, -70, -35]
      regionWeight: 16
      missionWeights:
!       STR_ALIEN_RESEARCH: 10
!       STR_ALIEN_HARVEST: 10
!       STR_ALIEN_ABDUCTION: 15
!       STR_ALIEN_INFILTRATION: 20
!       STR_ALIEN_BASE: 6
      missionZones:
        -
          - [334, 396.125, -66.125, -58.125]
--- 245,251 ----
        - [335, 60, -70, -35]
      regionWeight: 16
      missionWeights:
!        #STR_ALIEN_FLYBY: 1
      missionZones:
        -
          - [334, 396.125, -66.125, -58.125]
***************
*** 294,304 ****
        - [335, 55, -15, 0]
      regionWeight: 12
      missionWeights:
!       STR_ALIEN_RESEARCH: 8
!       STR_ALIEN_HARVEST: 8
!       STR_ALIEN_ABDUCTION: 8
!       STR_ALIEN_INFILTRATION: 8
!       STR_ALIEN_BASE: 19
      missionZones:
        -
          - [348.5, 384.5, -34.5, -25]
--- 296,302 ----
        - [335, 55, -15, 0]
      regionWeight: 12
      missionWeights:
!        #STR_ALIEN_FLYBY: 1
      missionZones:
        -
          - [348.5, 384.5, -34.5, -25]
***************
*** 341,351 ****
        - [5, 55, 0, 40]
      regionWeight: 8
      missionWeights:
!       STR_ALIEN_RESEARCH: 7
!       STR_ALIEN_HARVEST: 8
!       STR_ALIEN_ABDUCTION: 8
!       STR_ALIEN_INFILTRATION: 8
!       STR_ALIEN_BASE: 9
      missionZones:
        -
          - [7.125, 27, -0.375, 7.625]
--- 339,345 ----
        - [5, 55, 0, 40]
      regionWeight: 8
      missionWeights:
!        #STR_ALIEN_FLYBY: 1
      missionZones:
        -
          - [7.125, 27, -0.375, 7.625]
***************
*** 391,401 ****
        - [70, 90, -35, -5]
      regionWeight: 12
      missionWeights:
!       STR_ALIEN_RESEARCH: 5
!       STR_ALIEN_HARVEST: 5
!       STR_ALIEN_ABDUCTION: 6
!       STR_ALIEN_INFILTRATION: 10
!       STR_ALIEN_BASE: 8
      missionZones:
        -
          - [39.5, 53.125, -27.625, -16.125]
--- 385,391 ----
        - [70, 90, -35, -5]
      regionWeight: 12
      missionWeights:
!        #STR_ALIEN_FLYBY: 1
      missionZones:
        -
          - [39.5, 53.125, -27.625, -16.125]
***************
*** 444,454 ****
        - [105, 150, -50, -5]
      regionWeight: 15
      missionWeights:
!       STR_ALIEN_RESEARCH: 8
!       STR_ALIEN_HARVEST: 13
!       STR_ALIEN_ABDUCTION: 21
!       STR_ALIEN_INFILTRATION: 13
!       STR_ALIEN_BASE: 8
      missionZones:
        -
          - [90.125, 108.25, -36.625, -22.625]
--- 434,440 ----
        - [105, 150, -50, -5]
      regionWeight: 15
      missionWeights:
!        #STR_ALIEN_FLYBY: 1
      missionZones:
        -
          - [90.125, 108.25, -36.625, -22.625]
***************
*** 497,505 ****
        - [60, 180, -70, -50]
      regionWeight: 10
      missionWeights:
!       STR_ALIEN_RESEARCH: 6
!       STR_ALIEN_ABDUCTION: 4
!       STR_ALIEN_BASE: 9
      missionZones:
        -
          - [49.875, 80.125, -66.75, -46.5]
--- 483,489 ----
        - [60, 180, -70, -50]
      regionWeight: 10
      missionWeights:
!        #STR_ALIEN_FLYBY: 1
      missionZones:
        -
          - [49.875, 80.125, -66.75, -46.5]
***************
*** 540,550 ****
        - [105, 180, -5, 60]
      regionWeight: 10
      missionWeights:
!       STR_ALIEN_RESEARCH: 15
!       STR_ALIEN_HARVEST: 11
!       STR_ALIEN_ABDUCTION: 4
!       STR_ALIEN_INFILTRATION: 8
!       STR_ALIEN_BASE: 16
      missionZones:
        -
          - [129.625, 155.375, -1.875, 9.75]
--- 524,530 ----
        - [105, 180, -5, 60]
      regionWeight: 10
      missionWeights:
!        #STR_ALIEN_FLYBY: 1
      missionZones:
        -
          - [129.625, 155.375, -1.875, 9.75]
***************
*** 593,600 ****
        - [150, 180, -50, -5]
      regionWeight: 5
      missionWeights:
!       STR_ALIEN_RESEARCH: 15
!       STR_ALIEN_BASE: 27
      missionZones:
        -
          - [194.5, 225.625, -29.25, -15.25]
--- 573,579 ----
        - [150, 180, -50, -5]
      regionWeight: 5
      missionWeights:
!        #STR_ALIEN_FLYBY: 1
      missionZones:
        -
          - [194.5, 225.625, -29.25, -15.25]
***************
*** 650,653 ****
      areas:
        - [55, 70, -15, 10]
        - [70, 90, -5, 10]
!       - [55, 105, 10, 60]
\ No newline at end of file
--- 629,632 ----
      areas:
        - [55, 70, -15, 10]
        - [70, 90, -5, 10]
!       - [55, 105, 10, 60]
